configs/sandpack-contents/component-theming/modal.js

```diff
@@ -0,0 +1,129 @@
+module.exports = {
+  App: `import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  useDisclosure,
+  Button,
+  Box,
+  SimpleGrid,
+  IconButton,
+  useColorMode,
+} from "@chakra-ui/react";
+import { FaMoon, FaSun } from "react-icons/fa";
+
+
+export default function App() {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { toggleColorMode, colorMode } = useColorMode();
+
+  return (
+      <Box position="relative" h="100vh" p={12}>
+      <Button onClick={onOpen}>Open Modal</Button>
+
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Modal Title</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            Lorem ipsum dolor sit amet. Et corporis quisquam eum adipisci
+            impedit quo eius nisi est aspernatur vel veniam velit qui numquam
+            totam. Vel debitis sint ut culpa cupiditate a dolores voluptates ut
+            vero voluptatem non rerum aliquid qui sapiente possimus. Eum natus
+            voluptates hic galisum architecto et nobis incidunt ut odio ipsum
+            qui repudiandae voluptatem.
+          </ModalBody>
+
+          <ModalFooter>
+            <Button colorScheme="blue" mr={3} onClick={onClose}>
+              Close
+            </Button>
+            <Button variant="ghost">Secondary Action</Button>
+          </ModalFooter>
+        </ModalContent>
+      </Modal>
+      <IconButton
+        rounded="full"

```

Comment:
We should add a `aria-label="change theme"` to fix the issue shown in CodeSandbox.